integration/cri-o/crio_skip_tests.sh

```diff
@@ -5,6 +5,44 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# TODO: the list of files is hardcoded for now, as we don't have listed all tests
+# that need to be skipped, but ultimately we will want to use *.bats
+declare -a bats_files_list=(
```

Comment:
Hey @xiangchunfu , as per @littlejawa comment, now that #4434 is merged, when you rebase you will need to remove the .bats extension from the file list.

Reply:
Thanks, updated